Dockerfile

```diff
@@ -71,9 +70,10 @@ COPY . /app
 # We're installing here libexpat1, to upgrade the package to include a fix to 3 high CVEs. CVE-2024-45491,CVE-2024-45490,CVE-2024-45492
 RUN apt-get update \
     && apt-get install -y \
-       git \
-       apt-transport-https \
-       gnupg2 \
+    git \
+    apt-transport-https \
+    gnupg2 \
+    pandoc \
```

Comment:
`pypandoc` needs `pandoc` on the image to work?

Reply:
Yes. It is possible to download it from the code though if we prefer to do that.

Cf. https://pypi.org/project/pypandoc/ => `Installing pandoc via pypandoc`:

```py
pip install pypandoc
from pypandoc.pandoc_download import download_pandoc
# see the documentation how to customize the installation path
# but be aware that you then need to include it in the `PATH`
download_pandoc()
```